app/components/addRepository/add.repository.component.ts

```diff
@@ -69,16 +69,16 @@ export class AddRepositoryComponent {
   }
   prepareDontMissDND :  function() {
 
-      $(document.body).bind("dragover", function(e) {
-          e.preventDefault();
-          return false;
-      });
+      // $(document.body).bind("dragover", function(e) {
```

Comment:
Suggestion, either:
1.) delete these
2.) provide commentary as to why they are commented out.

Reply:
Oki i did write in in the PR . but you are right will write it in code also.